### DISCUSSION TITLE
Adding IMX390 Support

### DISCUSSION THREAD

**Discussion by markwu2001** [HimaxWiseEyePlus/Seeed_Grove_Vision_AI_Module_V2#47] [Q&A]
Hello! I'm curious if the team is working on [IMX390 support](https://www.seeedstudio.com/IMX390-2-12MP-118-H-FOV-4-lane-MIPI-CSI-Camera-p-5708.html). If not, are there tools the team can suggest for writing or porting camera drivers (maybe from linux kernel)? I would really like to use the IMX390 or IMX290/IMX327/IMX462 with this MCU due to its ultra small form factor :)

**Comment by richwriroo** [comment]
Likewise I would love to see this or get some guidance as to how to add support for these - specifically I have one of these https://www.amazon.com/dp/B0C3V6KHZT (imx462) and one of these https://www.uctronics.com/presale-arducam-2mp-ultra-low-light-starvis-imx327-motorized-ir-cut-camera-for-raspberry-pi.html

**Comment by CEGC2024** [comment]
Sensor manufactures are very secretive about their products. Out of all the manufacturers I have contacted, only two have being willing to communicate, share datasheets and application notes after signing an NDA. I suspect one issue is that Himax produce their own sensors, hence the likes of Sony are going to be reluctant to share the required information to make this happen.

I think an alternative solution in this case would be for a third party such as econ-systems or Uctronics/Arducam develop FPGA/ISP solution to bridge between the HX5638 and Sony/Omnivision, etc to enable use of the devices without disclosing sensitive information directly.
